pkg/kubelet/network/dns/dns.go

```diff
@@ -161,15 +161,15 @@ func (c *Configurer) CheckLimitsForResolvConf() {
 	f, err := os.Open(c.ResolverConfig)
 	if err != nil {
 		c.recorder.Event(c.nodeRef, v1.EventTypeWarning, "CheckLimitsForResolvConf", err.Error())
-		klog.V(4).Infof("CheckLimitsForResolvConf: " + err.Error())
+		klog.V(4).Infof("Check limits for resolv.conf failed: %v", err)
```

Comment:
since the event is now not matching the log file entry, perhaps log messages may be different. For example, here it will be 

```suggestion
		klog.V(4).Infof("Check limits for resolv.conf failed at file open: %v", err)
```

Reply:
Thanks for your review, fixed.